src/main/webapp/src/app/map/map.component.ts

```diff
@@ -1,12 +1,34 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, AfterViewInit, ViewChild, ElementRef } from 
+  '@angular/core';
+import { } from 'googlemaps';
 
 @Component({
   selector: 'app-map',
   templateUrl: `map.component.html`,
   styleUrls: ['./map.component.css']
 })
 
-export class MapComponent implements OnInit {
-  constructor() {  }
-  ngOnInit() {  }
+export class MapComponent implements AfterViewInit {
+
+  @ViewChild("mapContainer", { static: false })
+  mapContainer!: ElementRef;
+  map!: google.maps.Map;
+  lat = -34;
```

Comment:
Is this lat/lng somewhere in particular? Stick a comment here about why you've picked these.